prebid-server/developers/add-new-bidder-go.md

```diff
@@ -34,14 +41,36 @@ We ask that the first 6 letters of the name you choose be unique among the exist
 
 Throughout the rest of this document, substitute `{bidder}` with the name you've chosen.
 
+### Consider Your Geography
+
+Most bidders run their auction endpoints in multiple datacenters because their
+business is continental or global.
+
+Prebid Server is open source software that is run by many "host companies" that may be calling
```

Comment:
Instead of quoting host companies, add a link to either the glossary or a section where host companies are already described.